gridt/src/app/app.component.ts

```diff
@@ -36,6 +40,10 @@ export class AppComponent implements OnInit, OnDestroy{
     private auth: AuthService
   ) {
     this.initializeApp();
+    this.isLoggedIn$=this.router.events.pipe(
```

Comment:
Could you give the "=" some space to breath? Just put some spaces around it for readability's sake, please.